transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -152,11 +154,17 @@ public void initChannel(final Channel ch) {
                     @Override
                     public void run() {
                         pipeline.addLast(new ServerBootstrapAcceptor(
-                                ch, currentChildGroup, currentChildHandler, currentChildOptions, currentChildAttrs));
+                                ch, currentChildGroup, currentChildHandler, currentChildOptions, currentChildAttrs,
+                                extensions));
                     }
                 });
             }
         });
+        if (!extensions.isEmpty() && channel instanceof ServerChannel) {
+            for (ChannelInitializerExtension extension : extensions) {
+                extension.postInitializeServerListenerChannel((ServerChannel) channel);
```

Comment:
I wonder if we should handle exceptions which might be thrown here... Same is true for all the other methods.

Reply:
Good idea. I think we can only catch-and-log in this case.